manually_build_docs.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Restore fake and paket
+dotnet tool restore
+
+# Restore paket deps
+dotnet paket restore
```

Comment:
Nice! Perhaps something to have instead of the existing bootstrapped?

Reply:
That'd be better... want to focus on bootstrapping separately though...